### ISSUE TITLE
The command line tries to use the dart command

### ISSUE THREAD

**Issue by g123k** [celest-dev/celest#98]
Hi!

I would like to try celest, but as soon as the `start` command, I have the following error:

```
➜ celest start  
Enter a name for your project (androidmakers_schedule) androidmakers_schedule
⠧ Generating project... (0.6s)
⠧ Generating project... (1.4s)Failed to run `pub get`. Please run `/Users/g123k/fvm/versions/stable/bin/cache/dart-sdk/bin/dart pub get` manually in /Users/[...]/celest and try again.
```

And if I try the command, here is the error:

```
Because XXX depends on flutter_test from sdk which doesn't exist (the Flutter SDK is not available), version solving failed.

Flutter users should use `flutter pub` instead of `dart pub`.
```

**Comment by dnys1** [comment]
Hi @g123k 👋 

This is by design currently as we don't compile the Flutter SDK into your server code. Would `package:test` work for your use case instead of `flutter_test`?

**Comment by dnys1** [comment]
@g123k currently, the `dart` command in your PATH must point to the version from the Flutter SDK. Can you confirm by running `which dart` and update your PATH if it's pointing to a non-Flutter Dart (e.g. Homebrew)?

I'm working on a fix for this, so it should be a temporary measure.

**Comment by dnys1** [comment]
Okay, thanks for confirming. I'll look into it some more!

**Comment by dnys1** [comment]
This has been fixed in v0.3.10! You can run `celest upgrade` to get the changes.

Please let me know if you face any more issues.
